mirdata/download_utils.py

```diff
@@ -6,7 +6,7 @@
 import os
 import shutil
 import tarfile
-import urllib
+import urllib.request as urllib
```

Comment:
Could you explain why this change is needed? Thanks!

Reply:
In my local environment has an error. I fixed it. :-)